packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -126,6 +130,46 @@ function PreviewView() {
     }
   };
 
+  const MAX_RECORDING_TIME = 300;
```

Comment:
append units to this constant such that it is clear, e.g. `MAX_RECORDING_TIME_SEC`